### PR TITLE
Add Indonesian document

### PR DESCRIPTION
**Pull request by bagasf2002**
correct me if I'm wrong

### CODE REVIEW

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]
Incorrect link on line 52 & 72

**Comment by bagasf2002** [comment]
Check sar

**Comment by bagasf2002** [comment]
> Incorrect link on line 52, 72 & 107

Check sar

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]
Incorrect link on line 52, 72

**Comment by bagasf2002** [comment]
> Incorrect link on line 52, 72

Check sar

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]
Same

**Comment by bagasf2002** [comment]
> Same

Cek sar

File: docs/README-id.md
Comment (MlgmXyysd) [inline docs/README-id.md:1]: Same

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]
Incorrect link on line 52, 72

**Comment by MlgmXyysd** [comment]
Close due to no responding

**Pull request closed without merging**